lib/engine/step/g_1860/route.rb

```diff
@@ -74,10 +74,11 @@ def process_run_routes(action)
             train = route.train
             leased = ' '
             if train.owner && @game.train_owner(train) != entity
-              @game.game_error("Cannot run another corporation's train. refresh")
-              @game.game_error('Cannot run train that operated') if train.operated
+              raise GameError, "Cannot run another corporation's train. refresh"
             end
-            @game.game_error('Cannot run train twice') if trains[train]
+            raise GameError, 'Cannot run train that operated' if train.operated
```

Comment:
This one was intentional for rubocop (it pointed out that the `if train.operated` line could not be reached) but possibly a mistake -- @roseundy what is correct here?